src/index.html

```diff
@@ -662,7 +662,10 @@ <h2 class="card__title">Covid Regulations</h2>
                 Due to the current situation regarding COVID-19, we will apply sanitary and safety measures in order to
                 ensure the safest possible environment for every hacker at HackUPC. We will adhere to the regulations
                 imposed by the Generalitat de Catalunya and continuously update the information provided on this
-                <a href="https://web.gencat.cat/en/activem/restriccions-territorials/catalunya/" target="_blank"
+                <a
+                  rel="noopener noreferrer"
+                  href="https://web.gencat.cat/en/activem/restriccions-territorials/catalunya/"
```

Comment:
You can also add the `hreflang="en"` attribute.

```suggestion
                  href="https://web.gencat.cat/en/activem/restriccions-territorials/catalunya/"
                  hreflang="en"
```